docs/architecture.md

```diff
@@ -88,3 +88,8 @@ communicate with Kubernetes and where the additional types are stored:
 KubeVirt is deployed on top of a Kubernetes cluster.
 This means that you can continue to run your Kubernetes-native workloads next
 to the VMIs managed through KubeVirt.
+
+Furthormore: if you can run native workloads, and you have KubeVirt installed,
+you should be able to run VM-based workloads, too.
+For example, Application Operators should not require additional permissions
```

Comment:
Maybe we can write it this way:

```
For example, Application Operators should not require additional permissions to use cluster features for VMs, compared to using that feature with a plain Pod.
```
That would mention that this is meant in comparison with the permissions needed on a Pod.